app.js

```diff
@@ -1,134 +1,117 @@
-      newUl.appendChild(newLi);
-    }
-
-    var TotalLi = document.createElement('li');
-    TotalLi.innerText = 'Total' + ': ' + this.total + ' cookies';
-    newUl.appendChild(TotalLi);
-    body.appendChild(newUl);
-  };
+  storeArray.push(this);
 };
-
-//Table
-//table function
+console.log(storeArray);
 var tabFunction = function() {
   var table = document.createElement('table');
   body.appendChild(table);
-
-// table head
   var tHead = document.createElement('thead');
   table.appendChild(tHead);
   var headRow = document.createElement('tr');
   tHead.appendChild(headRow);
-
   var spacer = document.createElement('th');
   headRow.appendChild(spacer);
-
   for (var i = 0; i < storeHours.length; i++) {
     var tData = document.createElement('th');
     tData.innerText = storeHours[i];
     headRow.appendChild(tData);
   };
-
-  //table body
   var tBody = document.createElement('tbody');
   table.appendChild(tBody);
-
+  var tFoot = document.createElement('tfoot');
+  tFoot.id = 'bob';
+  table.appendChild(tFoot);
   var tabRow = document.createElement('tr');
   tBody.appendChild(tabRow);
-
   newTableRow(firstPike);
   newTableRow(seaTac);
   newTableRow(seaCenter);
   newTableRow(capHill);
   newTableRow(alki);
-
-//footer
-  var tFoot = document.createElement('tf');
-  body.appendChild(tFoot);
 };
 
 function newTableRow(store) {
   var tBody = document.getElementsByTagName('tbody')[0];
   var tabRow = document.createElement('tr');
   tBody.appendChild(tabRow);
-
   var rowName = document.createElement('th');
   tabRow.appendChild(rowName);
   rowName.innerText = store.name;
-
   for (var i = 0; i < storeHours.length; i++) {
     var tData = document.createElement('td');
     tData.innerText = store.salesArray[i];
     tabRow.appendChild(tData);
   };
 }
-//End Table
-
 tabFunction();
-
-//
 var form = document.getElementById('the-form');
+
 function createNewStore(event) {
   event.preventDefault();
   var name = event.target.elements.storeName;
   var minCust = event.target.elements.minCust;
   var maxCust = event.target.elements.maxCust;
   var avgCookieSale = event.target.elements.aveCookies;
-
   if (maxCust < minCust) {
     alert('The max number of customers should not be larger than the min number of customers');
   } else {
     var newStore = new Store(name.value, Math.floor(minCust.value), Math.floor(maxCust.value), avgCookieSale.value);
     newTableRow(newStore);
-
+    var tFoot = document.getElementById('bob');
+    var totalsRow = document.getElementById('shirley');
+    tFoot.removeChild(totalsRow);
+    hourlyTotals();
     form.reset();
   }
 }
 
+function hourlyTotals() {
+  var tFoot = document.getElementsByTagName('tfoot')[0];
+  var tabRow = document.createElement('tr');
+  tabRow.id = 'shirley';
+  tFoot.appendChild(tabRow);
+  var rowName = document.createElement('th');
+  tabRow.appendChild(rowName);
+  rowName.innerText = 'Total Sales';
+  for (h = 0; h < storeHours.length; h++) {
```

Comment:
Why do you use `h` here as your counter variable instead of the conventional `i`?